_posts/2024-05-14-homebrew-4.3.0.md

```diff
@@ -0,0 +1,104 @@
+  [APIs are private by default](https://github.com/Homebrew/brew/pull/16831).
+
+- Homebrew's code documentation on [rubydoc.brew.sh](https://rubydoc.brew.sh)
+  [includes Sorbet data from `.rbi` to provide more types](https://github.com/Homebrew/brew/pull/16906).
+
+- [`brew command`](https://github.com/Homebrew/brew/pull/17186),
+  [`brew shellenv` and `brew setup-ruby` are significantly faster](https://github.com/Homebrew/brew/pull/17188).
+
+- [When the GitHub token used by Homebrew requires more scopes the message will clarify these.](https://github.com/Homebrew/brew/pull/16633)
+- [`brew upgrade --overwrite` is a new flag similar to `brew install --overwrite` and `brew link --overwrite` to delete files that already exist in the prefix while linking.](https://github.com/Homebrew/brew/pull/16851)
+- [`brew install --display-times` also works with casks.](https://github.com/Homebrew/brew/pull/17052)
+- [Tap migrations can also perform renames.](https://github.com/Homebrew/brew/pull/16648)
+- [`HOMEBREW_GITHUB_API_TOKEN` supports more types of GitHub tokens.](https://github.com/Homebrew/brew/pull/17001)
+- [The `brew desc --eval-all` warning only applies to `brew desc --search`.](https://github.com/Homebrew/brew/pull/17102)
+- [`brew tap` no longer shows untapped taps with API support.](https://github.com/Homebrew/brew/pull/16766)
+- [`brew upgrade` no longer truncates some version numbers.](https://github.com/Homebrew/brew/pull/16959)
+- [@BrewTestBot can no longer provided approving reviews on Homebrew/brew.](https://github.com/Homebrew/brew/pull/16916)
+- [Formulae can optionally restrict network access in build/test/postinstall sandboxes.](https://github.com/Homebrew/brew/pull/17081)
+- [`HOMEBREW_TEMP` is used more consistently for temporary files](https://github.com/Homebrew/brew/pull/16749)
+- [`brew update` outputs a message whenever it is autoupdating to make clear what is causing the delay. Also, `brew update` will attempt to update all taps, not just those on GitHub.](https://github.com/Homebrew/brew/pull/16855)
+- [`brew install`/`upgrade`/`outdated` will more intelligently auto-update when specifying formulae/casks from third-party taps.](https://github.com/Homebrew/brew/pull/17179)
+- [`brew bump-formula` and `brew bump-cask-pr` refuse to bump packages that Homebrew's tooling already handles.](https://github.com/Homebrew/brew/pull/16750)
+- [`brew install --adopt` is more permissive and quicker if the bundle versions match.](https://github.com/Homebrew/brew/pull/16889)
+- [`brew uninstall` and `brew reinstall` will skip cask quit/signal directives.](https://github.com/Homebrew/brew/pull/16507)
+- [`brew info --json=v2` returns a Cask's bundle versions in `bundle_version` and `bundle_short_version` keys.](https://github.com/Homebrew/brew/pull/16826)
+- [`brew info` and `brew tap-info` provide more consistent output indicating if a package or tap is installed.](https://github.com/Homebrew/brew/pull/17076)
```

Comment:
Is it worth adding a note about the `olname` field being removed from `brew info --json` output?

```suggestion
- [`brew info --json=v2` returns a Cask's bundle versions in `bundle_version` and `bundle_short_version` keys.](https://github.com/Homebrew/brew/pull/16826)
- [`brew info --json=v2` output no longer includes the deprecated `oldname` key for formulae. Use the `oldnames` array instead.](https://github.com/Homebrew/brew/pull/17285)
- [`brew info` and `brew tap-info` provide more consistent output indicating if a package or tap is installed.](https://github.com/Homebrew/brew/pull/17076)
```